docs/source/technical_tutorials/working_with_r/access_cmr_r.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<p>Add comments</p>

<pre class="ql-syntax" spellcheck="false">library("reticulate") # to use maap-py python&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;
library("paws") # to access S3 files
library("ncdf4") # to read HDF4/netcdf files locally
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/482/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>